### DISCUSSION TITLE
note's history across devices

### DISCUSSION THREAD

**Discussion by halyph** [glushchenko/fsnotes#1369] [Q&A] [answered]
I have discovered that note's history is different across devices:

E.g. 

**MacOS** Version 5.4.4 (528)
![Screenshot 2022-06-15 at 16 39 07](https://user-images.githubusercontent.com/241654/173855322-7d0831a1-a634-4fb4-80e4-ef27f73c6710.png)

**iPhone** Version 5.0.2 (241)
![Open note](https://user-images.githubusercontent.com/241654/173855772-395df98e-7b76-4200-adff-36f35dc4c14b.png)

Q: Could you please confirm that it's done on purpose and each device tracks own note history? Also, interesting to know motivation for such design :-)

**Comment by glushchenko** [comment, marked as answer]
Copy from Telegram:

> Oleksandr, [24 Nov 2022, 19:34:09]:
> I plan to add git to the phone too

> It is technically difficult, there is no other reason
